cs/src/core/Index/FasterLog/FasterLogIterator.cs

```diff
@@ -97,43 +100,45 @@ internal unsafe FasterLogScanIterator(FasterLog fasterLog, BlittableAllocator<Em
         /// Async enumerable for iterator
         /// </summary>
         /// <returns>Entry, entry length, entry address</returns>
```

Comment:
Please add to comments what the fourth value in the tuple represents.

Also, I believe it is possible to add tuple member names right here, like so:

public IAsyncEnumerable<(byte[] aa, int bb, long cc, long dd)> GetAsyncEnumerable([EnumeratorCancellation] CancellationToken token = default)

Reply:
Updating as per comment.